rsmtool/rsmeval.py

```diff
@@ -382,7 +390,6 @@ def run_evaluation(config_file, output_dir):
                                               df_test_human_scores,
                                               subgroups,
                                               second_human_score_column,
-                                              exclude_zero_scores=exclude_zero_scores,
```

Comment:
It was removed because this flag is not used at all inside `run_prediction_analyses()`.